### PR TITLE
add atbash-cipher exercise

### PR DESCRIPTION
**Pull request by larshp**

### CODE REVIEW

**Comment by larshp** [comment]
@mbtools or @g-back any comments?

File: exercises/practice/atbash-cipher/.meta/zcl_atbash_cipher.clas.abap [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ENDCLASS.
+
+
+
+CLASS zcl_atbash_cipher IMPLEMENTATION.
+
+  METHOD decode.
+
+    plain_text = encode( cipher_text ).
+    REPLACE ALL OCCURENCES OF ` ` IN plain_text WITH ''.
```
Comment (g-back) [inline exercises/practice/atbash-cipher/.meta/zcl_atbash_cipher.clas.abap:25]:
```suggestion
    REPLACE ALL OCCURRENCES OF ` ` IN plain_text WITH ''.
```

**Review by g-back — COMMENTED** [review]
Tests are green, looks good I'd say

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**